### DISCUSSION TITLE
General feedback

### DISCUSSION THREAD

**Discussion by felixwiemuth** [felixwiemuth/SimpleReminder#15] [General]
Share your experiences with SimpleReminder.
How do you like the app? How do you use it?
How does it compare to other apps?

If you have ideas for improvement, first check the [planned features](https://github.com/felixwiemuth/SimpleReminder#planned-features), [issues](https://github.com/felixwiemuth/SimpleReminder) (planned features, feature requests) and the discussions in the [Ideas](https://github.com/felixwiemuth/SimpleReminder/discussions/categories/ideas) category. If there is no matching topic yet, you can start a new discussion.

**Comment by FaqatZubair1** [comment]
You have added a separate app icon for "add reminder". It should be a widget rather than a separate app icon as I personally use app drawer and it appears in my app drawer as a separate app. Honestly, I don't like it that way. Because these things are widgets that are ought to appear on home screen and not in app drawer.

**Reply by felixwiemuth** [reply]
Thanks for the feedback.
There is already a discussion about this here: #22 

Some points in short:
- We want to have a shortcut to directly open "Add Reminder", while we also want to be able to launch the main app directly. Therefore 2 launcher icons.
- There is also a point about having the second launcher icon visible (in the app list), so that users know it's an essential part of the app.
- Widgets are rather for more complex functionality. But they could probably be "misused" for this; not sure how complex it would be to add. Also it would be less obvious for users that this (essential) functionality exists.
- If you use a drawer which lists all apps to open apps, then this comes with the downside that it includes all apps. At best your drawer allows to hide some of them.
- SimpleReminder could maybe add an option to hide a launcher icon from the app list (and thus your drawer) but not sure whether adding this complexity is worth it.
- "App shortcuts", which appear when long-clicking on a launcher icon, could replace the need for a second launcher icon, with the downsides that:
  - It is less obvious for users that they exist and can be added to the home screen
  - They have an additional app icon in the bottom-right corner, so it's not possible to have the same app icon as now

These are the reasons why I currently tend to keep the two launcher icons.
